src/group/GroupCard.js

```diff
@@ -49,9 +51,9 @@ export default class GroupCard extends PureComponent<Props, State> {
     const { selected } = this.state;
 
     const user = users.find(x => x.email === email);
-
+    const merge = [...selected, user];
     this.setState({
-      selected: [...selected, user],
+      selected: merge,
```

Comment:
Why did you change that?

Reply:
@borisyankov because I am trying to add a User type. Currently, there are 35 errors related to adding a user type, the change is honestly not relevant to this PR because I am trying to fix all the errors and this is one of them. "selected" is of type User, and "user" is not. For this reason, flow is throwing an error. This is an approach I tried t resolve this issue but it persisted so that is the same as what was there before. I will remove it from this PR because it's not relevant here and if I am done then I'll send a separate PR for that. I boiled down the errors to 20 so hopefully, soon I will send a separate PR for this. I am still learning how to use Github that's why because I didn't intend to include it in this PR.